arroyo/environment.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import structlog
+from structlog.types import EventDict
+
+
+def add_severity_attribute(
+    logger: logging.Logger, method_name: str, event_dict: EventDict
+) -> EventDict:
+    """
+    Set the severity attribute for Google Cloud logging ingestion
+    """
+    event_dict["severity"] = event_dict["level"].upper()
+    del event_dict["level"]
+    return event_dict
+
+
+def setup_logging() -> None:
+    structlog.configure(
+        wrapper_class=structlog.stdlib.BoundLogger,
+        logger_factory=structlog.stdlib.LoggerFactory(),
+        processors=[
```

Comment:
How does this work with custom processors, such as our Sentry integration? Doesn't this mean that users can't configure the logger?